pcsx2/PAD/Linux/wx_dialog/dialog.cpp

```diff
@@ -540,7 +541,7 @@ void PADDialog::OnButtonClicked(wxCommandEvent& event)
 			}
 			m_pan_tabs[gamepad_id]->Refresh();
 			m_pan_tabs[gamepad_id]->Update();
-			config_key(gamepad_id, i);
+			key_captured = config_key(gamepad_id, i);
```

Comment:
You could move the declaration here, that way it will silence the codacy complaint.